clusters/build-clusters/01_cluster/ci/_origin-release-build/nodejs-16/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM registry.ci.openshift.org/openshift/nodejs-16:latest
+USER root
+ENV PATH=$PATH:/opt/rh/nodejs-16/root/usr/bin
+RUN mkdir -p /go
+RUN chmod g+xw -R /go
```

Comment:
Can you explain how this Dockerfile is used? We don't have any go code in the repo, so I wouldn't expect to need these lintes.

Reply:
> Can you explain how this Dockerfile is used? We don't have any go code in the repo, so I wouldn't expect to need these lintes.

Oops! Forgot to remove it! The change is not needed as confirmed by the release team.